apps/site/assets/js/algolia-result.js

```diff
@@ -114,7 +114,6 @@ function iconFromGTFSId(id) {
   if (toSubway) {
     return Icons.getFeatureIcon(toSubway);
   }
-  if (id === "Silver Line") return Icons.getFeatureIcon("bus");
```

Comment:
🤔 I wonder if this should return `Icons.getFeatureIcon("silver_line")`?

Reply:
Ah the id is `silver_line` now so the id can just be passed as the parameter like the other types.